pkg/users/users.js

```diff
@@ -165,11 +165,31 @@ async function getLogins() {
         console.warn("Unexpected error when getting logged in accounts", err);
     }
 
+    // shadow-utils passwd supports an --all flag which is lacking on RHEL and
+    // stable Fedora releases. Available at least on Fedora since
+    // shadow-utils-4.14.0-5.fc40 (currently known as rawhide).
+    const locked_users_map = {};
+    try {
+        const locked_statuses = await cockpit.spawn(["passwd", "-S", "--all"], { superuser: "require", environ: ["LC_ALL=C"] });
```

Comment:
I'd rather do `err: "message"` to avoid letting the error message go to the journal.

Reply:
Good suggestion!